initfiles/componentfiles/configxml/esp.xsl

```diff
@@ -166,6 +159,17 @@
                     </verify>
                 </EspProtocol>
             </xsl:if>
+            <xsl:if test="string($controlPort) != ''">
```

Comment:
Maybe instead of having the ability to turn it on, I think espcontrol is useful enough that it should default to "ON" and some default port.  But there should be a way of turning it off.  I.e. if controlPort is empty it defaults to 8010 (or maybe some other port)... but if the admin sets controlPort == 0 then we turn it off.